webapp/channels/src/components/view_user_group_modal/view_user_group_modal_header/view_user_group_modal_header.tsx

```diff
@@ -103,7 +103,7 @@ const ViewUserGroupModalHeader = ({
         if (permissionToJoinGroup) {
             return (
                 <button
-                    className='user-groups-create btn btn-md btn-primary'
```

Comment:
`user-groups-create` can probably be removed from `view_user_group_modal.scss`.

Reply:
Done.